app/src/Commands/Pack/ImportCommand.php

```diff
@@ -123,17 +123,21 @@ public function execute(InputInterface $input, OutputInterface $output) : int
                 ->withUrl($project_config->getRepositoryUrl())
             ;
 
-            if ($this->repo_manager->repoExists($repo)) {
-                $this->writer->error(
-                    $output,
-                    "Repository {$repo->getName()} already exists!",
-                    "Use <fg=gray>doil repo:list</> to see current installed repos"
-                );
+            if ($this->repo_manager->repoUrlExists($repo)) {
```

Comment:
As the new empty repo is generated as local by default, only **local** repositories are checked for the repository url. If the imported instance has a repository url that already exists globally, a new `<instance>_import` repository will still be created locally.

Wouldn't it be better to also check for existing **global** repositories and re-use these as well (e.g. the ILIAS repo)?

Reply:
done!